src/index.js

```diff
@@ -1,2 +1,37 @@
-import {searchUsers} from "./fetch.js";
+import {searchUsers} from "./fetch";
+import {sortUsers} from "./sortUsers"
+
+let $root = document.querySelector("#root");
+
+$root.innerHTML = `
+    <form id="seachForm">
+        <input id="searchField" name="searchField"/>
+        <button>search</button>
+    </form>
+    <button id="sortUsers">sort</button>
+    <div id="usersList"></div>
+`;
+
+let $seachForm = document.querySelector("#seachForm");
+let $searchField = $seachForm.querySelector("#searchField");
+let $usersList = document.querySelector("#usersList");
+let $sortUsers = document.querySelector("#sortUsers");
+
+$seachForm.addEventListener('submit', function(event) {
+    event.preventDefault();
+    let userName = $searchField.value;
+    searchUsers(userName)
+        .then(renderUsers);
+})
+
+function renderUsers(usersData) {
+    $usersList.innerHTML = usersData.map(el => `
+    <div class="User">
+        <img src=${el.avatar_url} height=100 alt="">
```

Comment:
1. always wrap attributes values with double quotes: `src="${el.avatar_url}"`
2. `alt=""` - should never be empty
3. `height=100` is very outdated property, use css class when you want to assign height/width etc.

Reply:
by the way, dont you have commas between users?